src/Slices/Settings/UI/Tabs/Configuration/Provider.tsx

```diff
@@ -8,10 +9,42 @@ interface Props {
   settings: EnvironmentSettings.EnvironmentSettings;
 }
 
+function reducer(
```

Comment:
Can you add a comment as to Why the reducer was needed ? For later reference, cause I already forgot parts of the explanation from last week x)

Reply:
Done :)